### PR TITLE
Setup wasm examples (Ethereum) + Workflow for compiling wasm + Generate Example Payload for UI

### PR DESCRIPTION
**Pull request by thala321**
Setup wasm examples (Ethereum):
- block: https://github.com/massbitprotocol/massbit-subgraphs/tree/master/index-timestamp  (**done**)
- event: https://github.com/massbitprotocol/massbit-subgraphs/tree/master/event-handler  (**done**)
- quickswap: https://github.com/QuickSwap/QuickSwap-subgraph  (**done**)

How to run (this will run 5 ethereum tests: 3 for .wasm and 2 for .so compiliation)
```
robot ethereum.robot
```

After running, Index Manager will receive the request for .WASM deployment

---
Generate examples payloads for UI by running the below command (supports WASM and SO projects) (**done**)
```
python3 create_exampe_json.py 
```

@NhutVu23 @anhhuy0501  can you help integrate this in the UI + modify create_example_json.py
POSTMAN: https://documenter.getpostman.com/view/1937585/Tzm8GFjr
Document: https://github.com/massbitprotocol/massbitprotocol/blob/6fa09d9572dd4fd14323bdb65ca8a91c9df9def5/code-compiler/README.md

### CODE REVIEW

**Comment by thala321** [comment]
Progress:

Screenshot from 2021-08-13 15-22-24

We are able to compile simple projects: Ethereum Event,  Ethereum Block.
But complex project like QuickSwap requires multiple mapping files upload which code-compiler doesn't support yet.
Also, we need Index Manager to support receiving .wasm file

**Comment by thala321** [comment]
Test result:
Substrate and Solana (.SO files)
image

Ethereum tests are failing for Rust 
Can you help checking @anhhuy0501

**Comment by thala321** [comment]
All the ethereum tests have passed, thanks a @anhhuy0501 

Screenshot from 2021-08-17 18-50-57

File: .gitignore [hunk 1]
```diff
@@ -47,5 +47,8 @@ output.xml
 report.html
 e2e-test/robot*
 e2e-test/bin*
-
 log/
+
```
Comment (anhhuy0501) [inline .gitignore:51]: Remove this code

File: code-compiler/compiler/so.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        threading.Thread.__init__(self)
+
+    def run(self):
+        cargo_codegen = CargoCodegen(self.generated_folder)
+        cargo_codegen.run()  # TODO: This still block the request
+
+        print("Compiling...")
+        try:
+            # Docker container doesn't know about cargo path so we need to use $HOME
+            output = subprocess.check_output(["$HOME/.cargo/bin/cargo build --release"], stderr=subprocess.STDOUT,
```
Comment (anhhuy0501) [inline code-compiler/compiler/so.py:63]: Todo: show output log in realtime

Comment (thala321) [reply code-compiler/compiler/so.py:63]: https://github.com/massbitprotocol/massbitprotocol/issues/161

**Review by anhhuy0501 — APPROVED** [review]
LGTM

**Pull request merged**